unittests/AST/ASTImporterTest.cpp

```diff
@@ -529,6 +529,16 @@ TEST(ImportType, ImportPackExpansion) {
 }
 
 
+TEST(ImportDecl, ImportRecordDeclInFuncParams) {
+  MatchVerifier<Decl> Verifier;
+  EXPECT_TRUE(
+        testImport(
+          "int declToImport(struct data_t{int a;int b;} *d){ return 0; }",
+          Lang_CXX, "", Lang_CXX, Verifier,
```

Comment:
lang_c ??

Reply:
In C mode, a warning is given which breaks the test (probably the problem of the test). It still tests the thing in CXX mode.

Reply:
This is strange. Is this something that might need a fix in the `testImport` code?

Reply:
Maybe. It needs to be investigated. I suggest opening a separate issue about it.